.vscode/settings.template.json

```diff
@@ -4,6 +4,11 @@
     // To use the locally built compiler, after 'npm run build':
     // "typescript.tsdk": "built/local"
 
+    "[typescript][javascript][json][jsonc]": {
+        // "editor.formatOnSave": true,
```

Comment:
This is a template, so I feel like it could be uncommented 🤷

Reply:
Ah, this came down to whether or not I thought people would want to do it on save by default or manually invoke it, but you're totally right.